app/models/opponent.rb

```diff
@@ -36,6 +36,26 @@ class Opponent < ApplicationRecord
     interested_party: 'Interested party'
   }.freeze
 
+  def self.dummy_opponent
+    Opponent.find_by(other_party_id: 'OPPONENT_00000001') || create_dummy_opponent
+  end
+
+  def self.create_dummy_opponent
+    Opponent.create(
```

Comment:
I suggest always using `create!` instead of `create` unless you are processing the result of `create` and showing the errors

Reply:
Agreed